DOCUMENTATION.rst

```diff
@@ -0,0 +1,285 @@
+
+****************************************
```

Comment:
Could you put all documentation inside the docs/ folder - if possible it would be great if you could use the sphinx package to organize the documentation (we use this for e.g glue-core). If you don't have time to learn sphinx, at least just put the doc files all in a docs/ folder.